### DISCUSSION TITLE
Following the readme, why doesn't it work in VSCode?

### DISCUSSION THREAD

**Discussion by richard-workpath** [muhammadsammy/tailwindcss-classnames#402] [Q&A] [answered]
I have generated the classnames file and imported it:

```typescript
import {classnames} from 'tailwindcss-classnames'

export const fullWidthContainer = classnames('container', 'w-full');

export const button = classnames('border-none', 'rounded-sm');
```

But the types are not recognized:

![Screenshot 2022-04-11 at 22 53 31](https://user-images.githubusercontent.com/102306208/162830225-77c9cd3e-f2b2-4069-8c4c-9d262f38c592.png)

**Comment by muhammadsammy** [comment, marked as answer]
This has been changed in v3 https://github.com/muhammadsammy/tailwindcss-classnames#whats-new-in-v3 , https://github.com/muhammadsammy/tailwindcss-classnames/issues/293

**Reply by madsnedergaard** [reply]
Is there a reason why the code examples in README is still using the previous syntax? Also, how do I know what category a class falls into - is there a way to let VSCode handle this? (I could look up the types every time, but that is not very nice or efficient).

**Reply by MaxJLerman** [reply]
I agree, I've been banging my head against a wall for two days because the code in the README doesn't work and I couldn't understand why for the life of me.
